src/sentry/integrations/github/integration.py

```diff
@@ -263,9 +263,13 @@ def get_commit_context(
         if not commitInfo:
             return None
         else:
+            committed_date = parse_datetime(commitInfo.get("committedDate")).astimezone(
+                timezone.utc
+            )
+
             return {
                 "commitId": commitInfo.get("oid"),
-                "committedDate": commitInfo.get("committedDate"),
+                "committedDate": committed_date,
```

Comment:
According to codecov, the block above is not being tested. Please look into it before we merge this.
<img width="934" alt="image" src="https://github.com/getsentry/sentry/assets/44410/15a15c72-4f25-407c-b7aa-e003c02620e9">